src/lib/web-rpde-query.js

```diff
@@ -28,11 +28,11 @@ class RPDEQuery {
     sql += " WHERE updated_at < ((now() at time zone 'utc') - interval '"+ this.dont_serve_data_until_seconds_old +" seconds') ";
     // Now if user has specified after, add those
     if (this.afterTimestamp && this.afterId) {
-       sql += " AND ( ( extract(epoch from updated_at)::int = $2 AND data_id > $3) OR (extract(epoch from updated_at)::int > $2)) ";
+       sql += " AND ( ( extract(epoch from updated_at)::int = $2 AND rpde_id > $3) OR (extract(epoch from updated_at)::int > $2)) ";
```

Comment:
All this file is the wrong table - if you look at line 25 it's normalised_data not raw_data so this shouldn't be changed